src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
 
-}
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('Parameters must be greater than 0');
+    }
+
+    if (
+      this.a + this.b <= this.c
+      || this.a + this.c <= this.b
+      || this.c + this.b <= this.a
 
-export class Circle {
+    ) {
+      throw new Error(`It is not possible
+       to construct a triangle with these sides`);
+    }
+  }
 
+  getArea(): number {
+    const p = (this.a + this.b + this.c) / 2;
+    const s = Math.sqrt(p * ((p - this.a) * (p - this.b) * (p - this.c)));
+
+    return +(s).toFixed(2);
```

Comment:
Let's create a function that returning `Math.floor(value * 100) / 100;` and use it everywhere